bitnami/nginx/Chart.yaml

```
@@ -34,4 +34,4 @@ maintainers:
 name: nginx
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/nginx
-version: 18.1.15
+version: 18.1.16
```

As you are adding new parameters, we need to use a new minor version

```suggestion
version: 18.2.0
```